tutorials/Coordinates/Coordinates.ipynb

```diff
@@ -0,0 +1,781 @@
+      "Usinging Coordinates to describe on-sky locations and match catalogs"
+     ]
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Lets start by considering a field around the picturesque Hickson Compact Group 7.  To do anything with this, we need to get an object that represents the coordinates of the center of this group.\n",
+      "\n",
+      "In Astropy, the most common object you'll work with for coordinates is `SkyCoord`.  A `SkyCoord` can be created most easily directly from angles as shown below.  It's also wise to explicitly specify the frame your coordinates are in, although this is not strictly necessary because the default is ICRS. \n",
+      "\n",
+      "(If you're not sure what ICRS is, it's basically safe to think of it as an approximation to an equatorial system at the J2000 equinox)."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord(9.81625*u.deg, 0.88806*u.deg, frame='icrs')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "SkyCoord will also accept string-formatted coordinates, although you'll need to give units if it's not obvious from the format."
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0h39m15.9s', '0d53m17.016s', frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "SkyCoord('0:39:15.9', '0:53:17.016', unit=(u.hour, u.deg), frame='icrs')"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "If the object you're interested in is in SIMBAD, you can also look it up directly from its name using the `SkyCoord.from_name()` [class method](https://julien.danjou.info/blog/2013/guide-python-static-class-abstract-methods). \n",
+      "\n",
+      "(Note that this requires an internet connection.  It's safe to skip if you don't have one, because we defined it above explicitly.)"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "hcg7_center = SkyCoord.from_name('HCG 7')\n",
+      "hcg7_center"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": []
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "This object we just created has various useful ways of accessing the information contained within it.  In particular, the ``ra`` and ``dec`` attributes are specialized ``Quantity`` objects that store angles and provide pretty representations of those angles and some useful attributes to quickly convert to common angle units:"
```

Comment:
Nope, that doesn't exist.  It might be better as a processing layer in the HTML conversion, though...?  That way it will get automatically run in deployment rather than depending on someone to remember to do it manually.